### DISCUSSION TITLE
DOTA 2 server 10Sept2023

### DISCUSSION THREAD

**Discussion by janpaengeng** [ValveSoftware/Dota2-Gameplay#11774] [Polls]
HELLO, 
dota2 is lagging for days now any time its gonna get fixed? not play dota2 rn its crashing so bad my last game was 10am. 10Sept2023 and everyone is crashing or lagging. Anyone else have this problem?

**Comment by mguedesgg** [comment]
Yes, mine is too!!!

**Comment by cedricchi** [comment]
man Dota 2 is dying thx to valve
10 years of manipulated garbage quality matchmaking
